src/main.rs

```diff
@@ -80,12 +93,11 @@ async fn main() {
         let num_peers = known_peers.len();
         let fingerprint = kaboodle.fingerprint().await;
         log::info!("== Peers: {} ({:08x})", num_peers, fingerprint);
-        let num_peers_to_print = usize::max_value(); // adjust this downward to only show a subset
-        for (peer, peer_state) in known_peers.iter() {
-            log::info!("+ {peer}:\t{peer_state}");
-        }
-        if num_peers > num_peers_to_print {
-            log::info!("+ ... and {} more", num_peers - num_peers_to_print);
+        for (peer, peer_info) in known_peers.iter() {
+            let identity = String::from_utf8(peer_info.identity.to_vec())
+                .map(|id| format!(" ({id})"))
+                .unwrap_or_default();
```

Comment:
It's a string in the demo app, but the intention is for it to be an opaque byte blob for Kaboodle. For the thing I imagine us building on top of it, it will probably be a small serialized struct with a UUID and a list of which Serval roles we're fulfilling (e.g. storage, job queue, etc.).